site_audit.drush.inc

```diff
@@ -98,17 +98,100 @@ function site_audit_drush_command() {
     ),
   );
 
+  $options_codebase = array(
```

Comment:
For all options, please standardize on `xxxx-xxxx` (do not mix underscore / dash).